.github/workflows/test.yml

```diff
@@ -37,12 +37,9 @@ jobs:
     - name: Check with Black
       run: |
           black . --line-length 120 --check 
-    - name: Lint with flake8
+    - name: Lint with pylint
       run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings.
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=120 --statistics
+        pylint urpatimeout
```

Comment:
We don't need to use any settings? The default want is enough?

Reply:
We don't have to specify anything, pylint will automatically look for .pylintrc file in the root folder and takes the configuration from there.